terraform/hydra-projects.tf

```diff
@@ -122,6 +122,48 @@ resource "hydra_jobset" "nixpkgs_cuda" {
   email_override      = ""
 }
 
+resource "hydra_jobset" "nixpkgs_cuda_stable" {
+  project     = hydra_project.nixpkgs.name
+  state       = "enabled"
+  visible     = true
+  name        = "cuda-stable"
+  type        = "legacy"
+  description = "Testing CUDA support. Come help the CUDA team! https://nixos.org/community/teams/cuda/"
+
+  nix_expression {
+    file  = "pkgs/top-level/release-cuda.nix"
+    input = "nixpkgs"
+  }
+
+  input {
+    name              = "nixpkgs"
+    type              = "git"
+    value             = "https://github.com/NixOS/nixpkgs.git nixos-24.05-small"
+    notify_committers = false
+  }
+
+  input {
+    name              = "officialRelease"
+    type              = "boolean"
+    value             = "false"
+    notify_committers = false
+  }
+
+  input {
+    name              = "supportedSystems"
+    type              = "nix"
+    value             = "[ \"x86_64-linux\" ]"
```

Comment:
```suggestion
    value             = "[ \"x86_64-linux\" \"aarch64-linux\" ]"
```

Reply:
Note that this currently targets sbsa instead of jetsons

Reply:
Just making sure we reach parity with unstable.

Reply:
> Note that this currently targets sbsa instead of jetsons

What needs to be changed for that, something here or in nixpkgs?

Reply:
Hydra does not control what gets built beyond the entrypoint that it uses, so the change has to be in nixpkgs.

Reply:
Okay, lets wait for that then.

Reply:
Yes, the change has to be on the nixpkgs side, we have to supply a different list of `cudaCapabilities` for the `aarch64-linux` package set. IIRC it seemed like it's not completely straightforward to do with `release.nix`. Just got to sit down for a moment and do it